qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -378,7 +380,18 @@ def tabopen(self, url=None, background=None, explicit=False):
             tabbed_browser = objreg.get('tabbed-browser', scope='window',
                                         window=window.win_id)
             return tabbed_browser.tabopen(url, background, explicit)
-        tab = webview.WebView(self._win_id, self)
+
+        if objreg.get('args').backend == 'webengine':
```

Comment:
The abstraction is leaking here too, I think. @Kingdread pointed out another spot below. In this case, maybe we could hide this behind `tab.create`, which would create the appropriate class of tab based on the backend (and handle the selective importing).